src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -60,84 +61,22 @@ def persist_flow_run(self, run_info: FlowRunInfo):
         self.queue.put(run_info)
 
 
-class HealthyEnsuredProcess:
-    def __init__(self, executor_creation_func, context):
-        self.process = None
-        self.input_queue = None
-        self.output_queue = None
-        self.is_ready = False
-        self._executor_creation_func = executor_creation_func
-        self.context = context
-
-    def start_new(self, task_queue: Queue):
-        input_queue = self.context.Queue()
-        output_queue = self.context.Queue()
-        self.input_queue = input_queue
-        self.output_queue = output_queue
-
-        # Put a start message and wait the subprocess be ready.
-        # Test if the subprocess can receive the message.
-        input_queue.put("start")
-
-        current_log_context = LogContext.get_current()
-        process = self.context.Process(
-            target=_process_wrapper,
-            args=(
-                self._executor_creation_func,
-                input_queue,
-                output_queue,
-                current_log_context.get_initializer() if current_log_context else None,
-                OperationContext.get_instance().get_context_dict(),
-            ),
-            # Set the process as a daemon process to automatically terminated and release system resources
-            # when the main process exits.
-            daemon=True,
-        )
+def format_current_process(process_name, pid, line_number: int):
```

Comment:
Non-critical

format_current_process -> format_current_process_info?

Generally, a process cannot be formated...

Reply:
fixed.
Renamed 'format current process info'